v2/sourcedb-to-spanner/src/test/resources/DataTypesIT/mysql-data-types.sql

```diff
@@ -260,6 +260,19 @@ CREATE TABLE tiny_text_pk_table (
    CONSTRAINT PRIMARY KEY (id(20))
 );
 
+CREATE TABLE tiny_text_pk_table (
+                                    id TINYTEXT,
```

Comment:
Seems like tiny text table sql is duplicated?

Reply:
Thanks. Fixed in current push.